plugins/hls-eval-plugin/hls-eval-plugin.cabal

```diff
@@ -65,7 +65,7 @@ library
     , ghc
     , ghc-boot-th
     , ghc-paths
-    , ghcide                >=1.5.0  && <1.6
+    , ghcide                >=1.5.0.1  && <1.6
```

Comment:
Oops, so then this wasn't a ghcide minor release, my bad. This is why we need http://github.com/kowainik/policeman !

Reply:
Yeah, i revised the .cabal in hackage as well, see: #2396